app/src/main/java/com/codesoom/assignment/dto/UserResponseData.java

```diff
@@ -0,0 +1,37 @@
+package com.codesoom.assignment.dto;
+
+import com.codesoom.assignment.domain.users.User;
+
+/**
+ * 회원 데이터 전송에 필요한 데이터 정의
```

Comment:
이렇게 바꾸는 것도 고민해볼 수 있겠죠.
```suggestion
 * 회원 데이터 전송에 필요한 데이터.
```
```suggestion
 * 회원 데이터 전송에 필요한 데이터를 정의합니다.
```